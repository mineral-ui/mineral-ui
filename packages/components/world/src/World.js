import React, { PureComponent } from 'react';

export default class World extends PureComponent {
  render() {
    return <span>World</span>;
  }
}
