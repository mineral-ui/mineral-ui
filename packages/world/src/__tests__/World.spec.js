import React from 'react';
import { shallow } from 'enzyme';
import World from '../World';

function renderWorld() {
  return shallow(<World />);
}

describe('World', () => {
  it('renders', () => {
    const world = renderWorld();

    expect(world.exists()).toEqual(true);
  });

  it('renders correctly', () => {
    const world = renderWorld();

    expect(world).toMatchSnapshot();
  });
});
