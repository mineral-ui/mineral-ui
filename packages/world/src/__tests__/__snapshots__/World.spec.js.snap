// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`World renders correctly 1`] = `
<World>
  <glamorous(span)>
    <span
      className="css-nil"
    >
      World
    </span>
  </glamorous(span)>
</World>
`;
